tests/tests.py

```diff
@@ -153,5 +154,59 @@ def test_translate_not_valid_phrase(self):
         current_result = vb.translate("asldkfj", "en", "ru")
         self.assertEqual(current_result, False)
 
+    def test_respond_as_dict_1(self):
+        data = json.loads('[{"text": "hummus", "seq": 0}]')
```

Comment:
@tenorz007 Agreed. I was confused on that bit. Thanks for clearing it up. Between, continuing from your last statement. I had places where the API broke when some new definition was added. Mocks would be the way to go then. 

Could you add a possible demo mock test in a different PR? Will like to see how you are approaching this. 

Thanks!